.vscodeignore

```diff
@@ -17,6 +17,7 @@ esbuild.config.js
 scripts/**
 .yarnrc
 .nvmrc
+node_modules
```

Comment:
I haven't seen any errors that indicate that we need to keep anything from node_modules so I added this back.